src/utils/sourceListHelper.js

```diff
@@ -14,16 +22,15 @@ const sourceListHelper = incident => {
 
   return (
     <div>
-      {!incident?.src || incident?.src === [] ? (
+      {!sources || sources === [] || !Array.isArray(sources) ? (
```

Comment:
Is there a reason to have
```jsx
!sources ||
```
in this conditional?

Reply:
I'm not sure, I didn't write it originally. I just changed the function from requiring the entire incident since it only accesses the src property. I added the isArray() check to pass a test that was failing.

Reply:
It doesn't really break anything to have it, it's just redundant as far as I can tell. So don't worry about it I guess.